src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -3416,6 +3416,11 @@ public static boolean streamEntireSSTables()
         return conf.stream_entire_sstables;
     }
 
+    public static long timeoutDelay()
```

Comment:
the pattern is to be `getTimeoutDelay`, should follow the pattern

Reply:
suggest something like `getStreamTransferTaskTimeout` - see other comment.